c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,87 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce) {
```

Comment:
Our original motivation to make a directive for the suggestions was to have a partials template to avoid having large parts of HTML in the JS code, right? Are you still planning to do that?